dspace/config/submission-forms.xml

```diff
@@ -49,16 +49,20 @@
 
         <form name="traditionalpageone">
             <row>
-                <field>
-                    <dc-schema>dc</dc-schema>
-                    <dc-element>contributor</dc-element>
-                    <dc-qualifier>author</dc-qualifier>
+                <relation-field>
+                    <relationship-type>isAuthorOfPublication</relationship-type>
+                    <search-configuration>personConfiguration</search-configuration>
                     <repeatable>true</repeatable>
-                    <label>Authors</label>
-                    <input-type>name</input-type>
-                    <hint>Enter the names of the authors of this item.</hint>
-                    <required></required>
-                </field>
+                    <label>Author</label>
+                    <hint>Add an author</hint>
+                    <linked-metadata-field>
+                        <dc-schema>dc</dc-schema>
+                        <dc-element>contributor</dc-element>
+                        <dc-qualifier>author</dc-qualifier>
+                        <input-type>name</input-type>
+                    </linked-metadata-field>
+                    <required>At least one author (plain text or relationship) is required</required>
+                </relation-field>
```

Comment:
@benbosman : One thing I'm just now realizing.  If Entities are *optional* in DSpace 7, shouldn't the default `submission-forms.xml`  eventually *not* depend on Entities?

That said, I'm fine with letting this into `master` for now, as it'll allow us to demo this on the demo sites.  But, we may want to create a ticket to comment out any Entity specific features from `submission-forms.xml`, so they are disabled by default out-of-the-box.